### DISCUSSION TITLE
Style in local !== style in production

### DISCUSSION THREAD

**Discussion by danilockthar** [vercel/next.js#15740] [Help]
Hi. Im having a little css difference between my local development and production. I expected the two looks the same like other projects i have with next. Maybe its something in the new updates, but the styles in production looks a little bit bigger than localhost and its breaking my design.

Here is the difference:
![gifbeeper](https://user-images.githubusercontent.com/37883725/89045036-36809780-d321-11ea-9ac8-11be23683535.gif)

Also in production, if i refresh it breaks my page, my page doesnt stay in the same place, it changes on every refresh, this is weird, im working with styled components, and after 10 projects with Nextjs and styled-components ,its the first time i have seen this css behaviour:
![ezgif-5-6729f970e622](https://user-images.githubusercontent.com/37883725/89045331-b9a1ed80-d321-11ea-83dd-dc843078d214.gif)

I tried to use CSSBaseline from Zeit to reset all my styles but it doesnt work, the change persist between my two stages.

**Comment by firdan-machda** [comment]
are you using material/mui ? try to add `StyledEngineProvider` like so 

```
import * as React from 'react';
import { StyledEngineProvider } from '@mui/material/styles';

export default function GlobalCssPriority() {
  return (
    <StyledEngineProvider injectFirst>
      {/* Your component tree. Now you can override MUI's styles. */}
    </StyledEngineProvider>
  );
}
```

[mui docs](https://mui.com/guides/interoperability/)

**Reply by lancejpollard** [reply]
I am getting this too, just using CSS modules and no external libraries. Well, I guess I am using font-awesome at some point. Any ideas, how did you resolve it?

**Comment by Luise8** [comment]
Any progress here?
